app/app/page.jsx

```diff
@@ -0,0 +1,25 @@
+// app/page.jsx
+import Header from "./components/Header";
+import MainPart from "./components/MainPart";
+import Search from "./components/Search";
+import About from "./components/About";
+import SignIn from "./components/SignIn";
+import OurTeam from "./components/OurTeam";
+import MemorableLines from "./components/MemorableLines";
+import Footer from "./components/Footer";
+import styles from "./page.module.css"; // You can style the page here
+
+export default function Home() {
+    return (
+        <div className={styles.container}>
+            <Header />
```

Comment:
This could be an extra component:
```
const AppLayoutContainer = ({ children }) => {
  return (
    <div className={styles.container}>
      <Header />
      {children}
      <Footer />
    </div>
  );
  
  export default function Home() {
    return (
      <AppLayoutContainer>
        <MainPart />
        <Search />
        ...
      </AppLayoutContainer>
    );
  }
}
```